lib/log_format.c

```diff
@@ -737,6 +761,35 @@ qb_vsnprintf_deserialize(char *string, size_t str_len, const char *buf)
 				type_longlong = QB_TRUE;
 			}
 			goto reprocess;
+		case 'z':
+			fmt[fmt_pos++] = *format;
+			format++;
+			if (sizeof(size_t) == sizeof(long long)) {
+				type_long = QB_FALSE;
```

Comment:
`QB_FALSE` assignments for `'z'` are superfluous (unlike in `'l'` case).
At least if we don't want to care about behavior that is undefined anyway :-)